CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [v1.16.1](https://github.com/fastly/go-fastly/releases/tag/v1.16.1) (2020-07-06)
+
+[Full Changelog](https://github.com/fastly/go-fastly/compare/v1.16.0...v1.16.1)
+
+**Bug fixes:**
```

Comment:
Is this meant to be a Bug fix? More like Notes?

Reply:
I don't see a way to generate a Notes section using the github-changelog-generator and when I remove all tags from both of these, it doesn't generate a section for the latest release in the changelog.

Originally, I added 'bug fix' to the origin CI PR because it addressed the staticcheck issues (not exactly bugs).